packages/client/src/components/form/FormFieldGenerator.tsx

```diff
@@ -617,7 +633,7 @@ const GeneratedInputField = React.memo<GeneratedInputFieldProps>(
       )
     }
 
-    if (fieldDefinition.type === HIDDEN) {
+    if (fieldDefinition.type === HIDDEN || fieldDefinition.type === HTTP) {
```

Comment:
Do we need to render anything?

Reply:
is it because if I do not add this in the condition by default the form field generator generates a text input field